dockstore-client/src/main/java/io/dockstore/common/FileProvisionUtil.java

```diff
@@ -69,27 +75,48 @@ private FileProvisionUtil() {
     }
 
     static boolean downloadFromVFS2(String path, Path targetFilePath, int threads) {
+
+        // An extension of the Http4sFileProvider class to allow for setting the cookie specification to something
```

Comment:
Suggest your comments be in JavaDoc format.

Reply:
I would also reference the GitHub issue here, so a developer looking at this for the first time would know why you are doing this. That's more of a personal preference of mine, so your call.